.github/workflows/test.yml

```diff
@@ -71,7 +71,7 @@ jobs:
       # - experimental: Whether the build is "allowed to fail".
       matrix:
         php: ['5.4', '5.5', '5.6', '7.0', '7.1', '7.2', '7.3', '7.4']
-        phpcs_version: ['3.5.5', 'dev-master']
+        phpcs_version: ['3.7.1', 'dev-master']
```

Comment:
The comment on line 67 can also be removed.

Reply:
I like having the contextual information handy, so I've updated the doc comment to make more sense, in b58668ab808fbdc46ad55aac58b78021ac8863b0.